compiler/luci/service/src/Nodes/CircleReshape.cpp

```diff
@@ -14,8 +14,36 @@
  * limitations under the License.
  */
 
+#include "luci/Service/CircleShapeInference.h"
+#include "Check.h"
+
+#include "CircleShapeInferenceHelper.h"
 #include "CircleCloneNode.h"
 
+#include <luci/Log.h>
+
+namespace
+{
+
+std::ostream &operator<<(std::ostream &os, const loco::TensorShape &tensor_shape)
+{
+  os << "[";
+  for (uint32_t r = 0; r < tensor_shape.rank(); ++r)
+  {
+    if (r)
+      os << ",";
+
+    if (tensor_shape.dim(r).known())
+      os << tensor_shape.dim(r).value();
+    else
+      os << "?";
+  }
+  os << "]";
+  return os;
+}
```

Comment:
(Not for this PR)

I find out that there are many `std::ostream &operator<<` overloadings for `TensorShape` in `luci/services`. 

* search `operator<<(std::ostream &os, const loco::TensorShape` in `compiler/luci/service/` 
	```c++
	compiler/luci/service/src/CircleShapeInference.cpp:
	  30  
	  31: std::ostream &operator<<(std::ostream &os, const loco::TensorShape &tensor_shape)
	  32  {
	
	compiler/luci/service/src/CircleShapeInferenceRule.cpp:
	  37  
	  38: std::ostream &operator<<(std::ostream &os, const loco::TensorShape &tensor_shape)
	  39  {
	
	compiler/luci/service/src/Validate.cpp:
	  33  
	  34: std::ostream &operator<<(std::ostream &os, const loco::TensorShape &tensor_shape)
	  35  {
	```

-------------------------------------

How about introduce new file name `compiler/luci/service/src/logHelper.h` and making one implementation?